Sources/App/CRP/SlackCommand+CRP.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+import Vapor
+import Stevenson
+
+extension SlackCommand {
+    static let crp = { (jira: JiraService, github: GitHubService) in
+        SlackCommand(
+            name: "crp",
+            help: """
+            Creates a ticket on the CRP board from specified release branch.
+
+            Parameters:
+            - project identifier (e.g: `ios`, `android`)
+            - branch: release branch name (e.g. `release/<version>`, `release/<app>/<version>`)
+
+            Example:
+            `/crp ios \(Option.branch):release/3.13.0`
+            """,
+            allowedChannels: ["ios-build"],
+            run: { metadata, request in
+                let components = metadata.text.components(separatedBy: " ")
+
+                guard let repo = components.first else {
+                    throw SlackService.Error.missingParameter(key: Option.repo)
+                }
+
+                guard let repoMapping = RepoMapping.all[repo.lowercased()] else {
+                    let all = RepoMapping.all.keys.joined(separator: "|")
+                    throw SlackService.Error.invalidParameter(
+                        key: Option.repo,
+                        value: repo,
+                        expected: all
+                    )
+                }
+
+                guard let branch = SlackCommand.branch(fromOptions: components) else {
+                    throw SlackService.Error.missingParameter(key: Option.branch)
+                }
+
+                let release = try github.makeGitHubRelease(
+                    repo: repoMapping.repository,
+                    branch: branch
+                )
+
+                return try github.changelog(for: release, request: request)
+                    .map { changelog in
+                        jira.makeCRPIssue(
+                            repoMapping: repoMapping,
+                            release: release,
+                            changelog: changelog
+                        )
+                    }
+                    .flatMap { issue in
+                        try jira.create(issue: issue, request: request)
+                    }
+                    .catchError(.capture())
+                    .map { issue in
+                        SlackResponse("""
+                            CRP Ticket \(issue.key) created.
+                            \(jira.baseURL)/browse/\(issue.key)
```

Comment:
If we change from `jira.host` to now using `jira.baseURL`, which now includes the port to use for the REST API (typically 443), not sure the generate URL will work? (no guarantee that the port to be used for the REST API is the same as for browsing the web UI)

Reply:
it's the same